main.tf

```
@@ -3,6 +3,15 @@ locals {
   service_account_email = var.create ? (
     length(google_service_account.lacework) > 0 ? google_service_account.lacework[0].email : ""
   ) : ""
+  service_account_name = length(var.service_account_name) > 0 ? (
+    var.service_account_name
+    ) : (
+    "lwsvc-${random_id.uniq.hex}"
+  )
```

Is there a reason for the formatting on this ternary statement?

```suggestion
  service_account_name = length(var.service_account_name) > 0 ? (
    var.service_account_name ) : "lwsvc-${random_id.uniq.hex}"
```

---

I have a VIM formatter for Terraform code. 😅 I'll accept this since it also looks good to me, I hope I don't put it back with the formatter later 😂